bin/cython_freeze

```diff
@@ -29,7 +29,20 @@ if options.output:
     old_stdout = sys.stdout
     sys.stdout = open(options.output, 'w')
 
-modules = [basename(splitext(x)[0]).replace('.', '_') for x in args]
+packages=[]
+modules=[]
+fullNames=[]
+
+for x in args:
+    moduleName=basename(splitext(x)[0]).replace('.', '_')
+    fullName=splitext(x)[0].replace(".", "_").replace("/", ".")
```

Comment:
I think it's worth giving a name to `splitext(x)[0]`. And also to `x`, e.g. `file_name`.

Reply:
Chngnes were made